<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | A.A. Research Lab </title> <meta name="author" content="A.A. Research Lab"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ainaadekunle.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">A.A.</span> Research Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/"> </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">News </a> </li> <li class="nav-item "> <a class="nav-link" href="/team/">Team </a> </li> <li class="nav-item "> <a class="nav-link" href="/join/">Join </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description"></p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="doi:10.1021/acs.jcim.3c01556" class="col-sm-8"> <div class="title">Cyclization Scaffolding for Improved Vaccine Immunogen Stability: Application to Tau Protein in Alzheimer’s Disease</div> <div class="author"> Shawn C. C. Hsueh, Mark Nijland, <em>Adekunle Aina</em>, and Steven S. Plotkin </div> <div class="periodical"> <em>Journal of Chemical Information and Modeling</em>, 2024 </div> <div class="periodical"> PMID: 38427576 </div> <div class="links"> <a href="https://doi.org/10.1021/acs.jcim.3c01556" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="doi:10.1021/acschemneuro.3c00007" class="col-sm-8"> <div class="title">De Novo Design of a β-Helix Tau Protein Scaffold: An Oligomer-Selective Vaccine Immunogen Candidate for Alzheimer’s Disease</div> <div class="author"> <em>Adekunle Aina</em>, Shawn C. C. Hsueh, Ebrima Gibbs, Xubiao Peng, Neil R. Cashman, and Steven S. Plotkin </div> <div class="periodical"> <em>ACS Chemical Neuroscience</em>, 2023 </div> <div class="periodical"> PMID: 37458595 </div> <div class="links"> <a href="https://doi.org/10.1021/acschemneuro.3c00007" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="aina2023prothon" class="col-sm-8"> <div class="title">PROTHON: A Local Order Parameter-Based Method for Efficient Comparison of Protein Ensembles</div> <div class="author"> <em>Adekunle Aina</em>, Shawn C. C. Hsueh, and Steven S. Plotkin </div> <div class="periodical"> <em>Journal of Chemical Information and Modeling</em>, 2023 </div> <div class="periodical"> PMID: 37178169 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acs.jcim.3c00145" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="/assets/pdf/PROTHON-A_Local_Order_Parameter-Based_Method_for_Efficient_Comparison_of_Protein_Ensembles.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p> The comparison of protein conformational ensembles is of central importance in structural biology. However, there are few computational methods for ensemble comparison, and those that are readily available, such as ENCORE, utilize methods that are sufficiently computationally expensive to be prohibitive for large ensembles. Here, a new method is presented for efficient representation and comparison of protein conformational ensembles. The method is based on the representation of a protein ensemble as a vector of probability distribution functions (pdfs), with each pdf representing the distribution of a local structural property such as the number of contacts between Cβ atoms. Dissimilarity between two conformational ensembles is quantified by the Jensen–Shannon distance between the corresponding set of probability distribution functions. The method is validated for conformational ensembles generated by molecular dynamics simulations of ubiquitin, as well as experimentally derived conformational ensembles of a 130 amino acid truncated form of human tau protein. In the ubiquitin ensemble data set, the method was up to 88 times faster than the existing ENCORE software, while simultaneously utilizing 48 times fewer computing cores. We make the method available as a Python package, called PROTHON, and provide a GitHub page with the Python source code at https://github.com/PlotkinLab/Prothon. </p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="hsueh2022optimizing" class="col-sm-8"> <div class="title">Optimizing Epitope Conformational Ensembles Using α-Synuclein Cyclic Peptide “Glycindel” Scaffolds: A Customized Immunogen Method for Generating Oligomer-Selective Antibodies for Parkinson’s Disease</div> <div class="author"> Shawn C. C. Hsueh, <em>Adekunle Aina</em>, Andrei Yu. Roman, Neil R. Cashman, Xubiao Peng, and Steven S. Plotkin </div> <div class="periodical"> <em>ACS Chemical Neuroscience</em>, 2022 </div> <div class="periodical"> PMID: 35840132 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acschemneuro.1c00567" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p> Effectively presenting epitopes on immunogens, in order to raise conformationally selective antibodies through active immunization, is a central problem in treating protein misfolding diseases, particularly neurodegenerative diseases such as Alzheimer’s disease or Parkinson’s disease. We seek to selectively target conformations enriched in toxic, oligomeric propagating species while sparing the healthy forms of the protein that are often more abundant. To this end, we computationally modeled scaffolded epitopes in cyclic peptides by inserting/deleting a variable number of flanking glycines (“glycindels”) to best mimic a misfolding-specific conformation of an epitope of α-synuclein enriched in the oligomer ensemble, as characterized by a region most readily disordered and solvent-exposed in a stressed, partially denatured protofibril. We screen and rank the cyclic peptide scaffolds of α-synuclein in silico based on their ensemble overlap properties with the fibril, oligomer-model and isolated monomer ensembles. We present experimental data of seeded aggregation that support nucleation rates consistent with computationally predicted cyclic peptide conformational similarity. We also introduce a method for screening against structured off-pathway targets in the human proteome by selecting scaffolds with minimal conformational similarity between their epitope and the same solvent-exposed primary sequence in structured human proteins. Different cyclic peptide scaffolds with variable numbers of glycines are predicted computationally to have markedly different conformational ensembles. Ensemble comparison and overlap were quantified by the Jensen–Shannon divergence and a new measure introduced here, the embedding depth, which determines the extent to which a given ensemble is subsumed by another ensemble and which may be a more useful measure in developing immunogens that confer conformational selectivity to an antibody. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="hsueh2022ensemble" class="col-sm-8"> <div class="title">Ensemble Generation for Linear and Cyclic Peptides Using a Reservoir Replica Exchange Molecular Dynamics Implementation in GROMACS</div> <div class="author"> Shawn C.C. Hsueh, <em>Adekunle Aina</em>, and Steven S. Plotkin </div> <div class="periodical"> <em>The Journal of Physical Chemistry B</em>, 2022 </div> <div class="periodical"> PMID: 36410027 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acs.jpcb.2c05470" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p> The profile of shapes presented by a cyclic peptide modulates its therapeutic efficacy and is represented by the ensemble of its sampled conformations. Although some algorithms excel at creating a diverse ensemble of cyclic peptide conformations, they seldom address the entropic contribution of flexible conformations and often have significant practical difficulty producing an ensemble with converged and reliable thermodynamic properties. In this study, an accelerated molecular dynamics (MD) method, namely, reservoir replica exchange MD (R-REMD or Res-REMD), was implemented in GROMACS ver. 4.6.7 and benchmarked on two small cyclic peptide model systems: a cyclized furin cleavage site of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike (cyclo-(CGPRRARSG)) and oxytocin (disulfide-bonded CYIQNCPLG). Additionally, we also benchmarked Res-REMD on alanine dipeptide and Trpzip2 to demonstrate its validity and efficiency over REMD. For Trpzip2, Res-REMD coupled with an umbrella-sampling-derived reservoir generated similar folded fractions as regular REMD but on a much faster time scale. For cyclic peptides, Res-REMD appeared to be marginally faster than REMD in ensemble generation. Finally, Res-REMD was more effective in sampling rare events such as trans to cis peptide bond isomerization. We provide a GitHub page with the modified GROMACS source code for running Res-REMD at https://github.com/PlotkinLab/Reservoir-REMD. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Gibbs2022rational" class="col-sm-8"> <div class="title">Rational Generation of Monoclonal Antibodies Selective for Pathogenic Forms of Alpha-Synuclein</div> <div class="author"> Ebrima Gibbs, Beibei Zhao, Andrei Roman, Steven S. Plotkin, Xubiao Peng, Shawn C. C. Hsueh, <em>Adekunle Aina</em>, Jing Wang, Clay Shyu, Calvin K. Yip, Sung-Eun Nam, Johanne M. Kaplan, and Neil R. Cashman </div> <div class="periodical"> <em>Biomedicines</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3390/biomedicines10092168" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Misfolded toxic forms of alpha-synuclein (α-Syn) have been implicated in the pathogenesis of synucleinopathies, including Parkinson’s disease (PD), dementia with Lewy bodies (DLB), and multiple system atrophy (MSA). The α-Syn oligomers and soluble fibrils have been shown to mediate neurotoxicity and cell-to-cell propagation of pathology. To generate antibodies capable of selectively targeting pathogenic forms of α-Syn, computational modeling was used to predict conformational epitopes likely to become exposed on oligomers and small soluble fibrils, but not on monomers or fully formed insoluble fibrils. Cyclic peptide scaffolds reproducing these conformational epitopes exhibited neurotoxicity and seeding activity, indicating their biological relevance. Immunization with the conformational epitopes gave rise to monoclonal antibodies (mAbs) with the desired binding profile showing selectivity for toxic α-Syn oligomers and soluble fibrils, with little or no reactivity with monomers, physiologic tetramers, or Lewy bodies. Recognition of naturally occurring soluble α-Syn aggregates in brain extracts from DLB and MSA patients was confirmed by surface plasmon resonance (SPR). In addition, the mAbs inhibited the seeding activity of sonicated pre-formed fibrils (PFFs) in a thioflavin-T fluorescence-based aggregation assay. In neuronal cultures, the mAbs protected primary rat neurons from toxic α-Syn oligomers, reduced the uptake of PFFs, and inhibited the induction of pathogenic phosphorylated aggregates of endogenous α-Syn. Protective antibodies selective for pathogenic species of α-Syn, as opposed to pan α-Syn reactivity, are expected to provide enhanced safety and therapeutic potency by preserving normal α-Syn function and minimizing the diversion of active antibody from the target by the more abundant non-toxic forms of α-Syn in the circulation and central nervous system.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="seifi2021structural" class="col-sm-8"> <div class="title">Structural fluctuations and mechanical stabilities of the metamorphic protein RfaH</div> <div class="author"> Bahman Seifi, <em>Adekunle Aina</em>, and Stefan Wallin </div> <div class="periodical"> <em>Proteins: Structure, Function, and Bioinformatics</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/https://doi.org/10.1002/prot.26014" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract RfaH is a compact two-domain bacterial transcription factor that functions both as a regulator of transcription and an enhancer of translation. Underpinning the dual functional roles of RfaH is a partial but dramatic fold switch, which completely transforms the  50-amino acid C-terminal domain (CTD) from an all-α state to an all-β state. The fold switch of the CTD occurs when RfaH binds to RNA polymerase (RNAP), however, the details of how this structural transformation is triggered is not well understood. Here we use all-atom Monte Carlo simulations to characterize structural fluctuations and mechanical stability properties of the full-length RfaH and the CTD as an isolated fragment. In agreement with experiments, we find that interdomain contacts are crucial for maintaining a stable, all-α CTD in free RfaH. To probe mechanical properties, we use pulling simulations to measure the work required to inflict local deformations at different positions along the chain. The resulting mechanical stability profile reveals that free RfaH can be divided into a “rigid” part and a “soft” part, with a boundary that nearly coincides with the boundary between the two domains. We discuss the potential role of this feature for how fold switching may be triggered by interaction with RNAP.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="aina2017multisequence" class="col-sm-8"> <div class="title">Multisequence algorithm for coarse-grained biomolecular simulations: Exploring the sequence-structure relationship of proteins</div> <div class="author"> <em>Adekunle Aina</em>, and Stefan Wallin </div> <div class="periodical"> <em>The Journal of Chemical Physics</em>, Sep 2017 </div> <div class="periodical"> 095102 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1063/1.4986933" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We consider a generalized-ensemble algorithm for coarse-grained simulations of biomolecules which allows the thermodynamic behavior of two or more sequences to be determined in a single multisequence run. By carrying out a random walk in sequence space, the method also enhances conformational sampling. Escape from local energy minima is accelerated by visiting sequences for which the minima are more shallow or absent. We test the method on an intermediate-resolution coarse-grained model for protein folding with 3 amino acid types and explore the potential for a large-scale coverage of sequence space by applying the method to sets of more than 1000 sequences. The resulting thermodynamic data are used to analyze the structures and stability properties of sequences covering the space between folds with different secondary structures.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 A.A. Research Lab. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> </body> </html>